mod_reforged/hooks/skills/actives/root_skill.nut

```diff
@@ -0,0 +1,21 @@
+::mods_hookExactClass("skills/actives/root_skill", function(o) {
```

Comment:
This is not the correct way. Look at our hook on rally_skill to see how to implement a better cooldown system. Then when you add the skill to the entity you specify the cooldown via `::MSU.new` (for this see how we add rally_skill to brigand_leader)